README.md

```diff
@@ -18,7 +46,25 @@ free and open source deep learning models.
 
 ## Run Locally
 
-### Deploy the Model
+#### Start the Model API
+
+1. [Deploy the Model](#1-deploy-the-model)
+2. [Experimenting with the API (Optional)](#2-experimenting-with-the-api-optional)
```

Comment:
Consistency: "Experiment with ..."

Reply:
That's an interesting differentiation but not likely one you'd get any tech writer to buy in. Imagine a translator would have to decide whether something is optional based on that rule - the results could be totally unpredictable. Take a look at some (random) official IBM documentation, such as https://console.bluemix.net/docs/containers/cs_firewall.html#firewall_bx 
Note that all sections start with a verb ending in "ing"). Each step starts with a command (Log in, get, retrieve, ...) and optional steps are preceded by "Optional" (see the first step 7 in above doc link).

Reply:
Fair enough, I'll change them